build.gradle

```diff
@@ -107,6 +108,8 @@ configurations {
     compile.exclude  module: "tomcat-jdbc"
 }
 
+ext ['spring-security.version'] = '5.3.0.RELEASE' // pinned due to CVE-2018-1258 in 5.2.2.RELEASE brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
```

Comment:
As @petergphillips mentioned on my PR, `5.3.0.RELEASE` is still affected by `CVE-2018-1258` (I still need to go back and change the comment, sorry)

Reply:
As he said, no harm in updating the version, but the comment is misleading since it doesn't resolve the CVE